resources/aws/bucket.go

```diff
@@ -8,13 +8,18 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 	microerror "github.com/giantswarm/microkit/error"
 	"github.com/juju/errgo"
+
+	"github.com/giantswarm/aws-operator/resources"
 )
 
 type Bucket struct {
 	Name string
 	AWSEntity
 }
 
+// Implement ReusableResource.
+var _ = resources.ReusableResource(&Bucket{})
```

Comment:
Is this better than just declaring `bucket := resources.ReusableResource` like you were doing before?

Also, is it idiomatic Go?